build.gradle

```diff
@@ -118,6 +121,12 @@ dependencies {
     compile 'org.apache.logging.log4j:log4j-api:2.6.2'
     compile 'org.apache.logging.log4j:log4j-core:2.6.2'
 
+    // need to use snapshots as the stable version is from 2013 and doesn't support v1.0.1 CitationStyles
+    compile 'org.citationstyles:styles:1.0.1-SNAPSHOT'
```

Comment:
Please also follow https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md#when-adding-a-library

Is there an issue asking for a release of the library? - If yes, please link it here and open an issue at jabref/issues linking that issue. If no, please either contact the maintainers directly or open an issue at their repo. Also open an issue at jabref/issues.